### PR TITLE
fix(data dictionary ): added descriptions

### PR DESCRIPTION
**Pull request by homelessbirds**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/data-dictionary/events/NrAiIncident/closeCause.md [hunk 1]
```diff
@@ -7,4 +7,19 @@ events:
 
 If applicable, what caused the incident to close.
 
-Available values: `CONDITION_DELETED`, `POLICY_DELETED`, `EVALUATOR`, `EXPIRED`, `CONDITION_MODIFIED`, `LOSS_OF_SIGNAL`, `USER`, `TARGET_REMOVED`, `INCIDENT_WORKFLOW_INTEGRATION`, and `CONDITION_DISABLED`.
+Available values: 
+`CONDITION_DELETED`: condition was deleted.
+`CONDITION_DISABLED` condition was disabled.
+`CONDITION_MODIFIED` condition was [edited](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#evaluation-resets)
+`EVALUATOR`: natural close because the metric has ceased to breach the threshold.
```
Comment (paperclypse) [inline src/data-dictionary/events/NrAiIncident/closeCause.md:14]: Think it'd be nice if this had more active voice

File: src/data-dictionary/events/NrAiIncident/closeCause.md [hunk 2]
```diff
@@ -7,4 +7,19 @@ events:
...
+`EXPIRED`:  incident closed due to TTL.
+`INCIDENT_WORKFLOW_INTEGRATION`: a third party closed the issue or incident through an API call.
+`LABEL_BINDINGS_CHANGED `:  label was removed from the target, causing it to no longer be covered by the condition. This only applies to APM/Mobile/Browser/Infra conditions.
```
Comment (paperclypse) [inline src/data-dictionary/events/NrAiIncident/closeCause.md:17]: I'd change "Infra" to "Infrastructure"

File: src/data-dictionary/events/NrAiIncident/closeCause.md [hunk 3]
```diff
@@ -7,4 +7,19 @@ events:
...
+`LOSS_OF_SIGNAL`: signal was lost, and incident was closed based on LoS settings.
```
Comment (paperclypse) [inline src/data-dictionary/events/NrAiIncident/closeCause.md:18]: I'd ditch the acronym in favor of "loss of signal"

**Review by paperclypse — DISMISSED** [review]
Looks good @homelessbirds. I left some feedback on some minor issues.

File: src/data-dictionary/events/NrAiIncident/closeCause.md (hunk 3 above)
Comment (homelessbirds) [reply src/data-dictionary/events/NrAiIncident/closeCause.md]: done!

File: src/data-dictionary/events/NrAiIncident/closeCause.md (hunk 1 above)
Comment (homelessbirds) [reply src/data-dictionary/events/NrAiIncident/closeCause.md:14]: done!

File: src/data-dictionary/events/NrAiIncident/closeCause.md (hunk 2 above)
Comment (homelessbirds) [reply src/data-dictionary/events/NrAiIncident/closeCause.md]: done!

**Pull request merged**